### ISSUE TITLE
Cannot view content of QVariant

### ISSUE THREAD

**Issue by borro0** [tonka3000/vscode-qt-tools#43]
I've setup a qt5.natvis file to see some qt debug info.

For QString and QVector it works.

However for QVariant, it doesn't show anything.
I'm in linux using gdb.

![image](https://user-images.githubusercontent.com/15669405/105716201-cb55fd80-5f1e-11eb-960e-f7a7db35ebd1.png)

**Comment by tonka3000** [comment]
Which natvis file do you use?

**Comment by borro0** [comment]
https://github.com/inviwo/inviwo/blob/master/tools/natvis/qt5.natvis

Op ma 25 jan. 2021 18:51 schreef Michael Aigner <notifications@github.com>:

**Comment by tonka3000** [comment]
I often use the official one https://code.qt.io/cgit/qt-labs/vstools.git/plain/src/qtvstools/qt5.natvis.xml, but this is the same file as you linked (without the namespace vars).

I just inject the natvis files to `launch.json` file (replace the namespace macro correclty) and do no special handling inside my extension. VSCode just use the C++ extension from microsoft to read/use the natvis file, so it is maybe a good idea to file an issue on their bug tracker https://github.com/microsoft/vscode-cpptools/issues . I've read in the past that the natvis file are using a complete different impl. on Linux/Mac as on windows. On windows it works most of the time normally. On my Mac it works partly.

**Comment by pktiuk** [comment]
@tonka3000   

Why do you use only such a basic natvis file in this extension instead of including a proper one from another project? (of course with proper FOSS license)

**Comment by Ashark** [comment]
> @tonka3000
> 
> Why do you use only such a basic natvis file in this extension instead of including a proper one from another project? (of course with proper FOSS license)

Yeah, I think there would not be a problem to include one file under different license.
